client/src/components/layout/TopBar.js

```diff
@@ -14,11 +15,21 @@ const TopBar = ({ user }) => {
       </Link>
     </li>,
   ];
-
+  let userPage
+  if (user) {
+    userPage = (
+      <li>
+        <Link to={`/user/page`}>Profile page</Link>
+      </li>
+    )
+  } else {
+    userPage = null
+  }
   const authenticatedListItems = [
+    userPage,
     <li key="sign-out">
       <SignOutButton />
-    </li>,
+    </li>
```

Comment:
You can do all of this with a ternary on a single line (29). But you dont even need the ternary. Literally just put the link into authenticatedListItems, it already only display if there is a logged in user